Project/VetAppointmentAPI/VetAppointmentAPI/VetAppointment.WebAPI/Controllers/UsersController.cs

```diff
@@ -73,6 +77,20 @@ private async Task<IActionResult> GetParsedUserData(User? user)
             return Ok(userDto);
         }
 
+        // POST api/<UsersController>
```

Comment:
We have a register method so there shouldn't be an exposed POST route for users.

Reply:
my bad :( gonna delete right away